assets/src/application/ui/layout/confirmDelete/ConfirmDelete.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Popconfirm } from 'antd';
+import { PopconfirmProps } from 'antd/lib/popconfirm';
+import { __ } from '@wordpress/i18n';
+
+const ConfirmDelete: React.FC<Partial<PopconfirmProps>> = ({ children, ...props }) => {
+	return (
+		<Popconfirm title={__('Are you sure delete it?')} okText={__('Yes')} cancelText={__('No')} {...props}>
```

Comment:
this will be the same all the time?

Reply:
It can be overridden

Reply:
I was/am used to a more explicit style, but with TS that's isn't that relevant